client/blocks/plugins-update-manager/index.tsx

```diff
@@ -102,7 +104,7 @@ export const PluginsUpdateManager = ( props: Props ) => {
 							onClick={ onCreateNewSchedule }
 							disabled={ ! canCreateSchedules }
 						>
-							Create a new schedule
+							{ translate( 'Set up a new schedule' ) }
```

Comment:
```suggestion
							{ translate( 'Add new schedule' ) }
```
Similar to "Add new post" in https://wordpress.com/posts/SITE